### PR TITLE
Fixed #902 -- Updated to Django==2.2.3

### LINKED ISSUE #902: Update Django to version 2.2
I think wold be good to update Django to the latest version 2.2 LTS and other outdated requirements.

### PR DESCRIPTION
**Pull request by pauloxnet**
Fix #902

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 -r common.txt
 
 watchdog==0.9.0
-transifex-client
+transifex-client==0.13.6
```
[the remark this answers is not in the document]
Comment (pauloxnet) [reply requirements/dev.txt]: I'm going to rebase my PR and split update into separate commits.

[the remark this answers is not in the document]
Comment (pauloxnet) [reply requirements/dev.txt]: In version 0.13.5 for a not clear reason they pinned some transifex-client dependencies to very specific version
https://github.com/transifex/transifex-client/commit/14a1a55c55fe66b1dee1fdfc4f2b1134c3c1af5d

I'm going to use the version 0.13.4 to have the more recent version of the library without the conflict for the pinned dependencies.

**Comment by pauloxnet** [comment]
@felixxm the rebase process have taken some commits I dont't want to be part of this PR.

Can you cherry-pick only the latest commit related to requests updates ?

**Comment by felixxm** [comment]
@pauloxnet Np, I resolved this.

**Comment by pauloxnet** [comment]
Thanks.

**Comment by felixxm** [comment]
I moved updating `django-money` to the #924 because it requires changes in `foundation.models` (see falling tests).

**Comment by pauloxnet** [comment]
Thanks for fixing it.

**Comment by felixxm** [comment]
Partly merged in 683e5570a1362b8a7b650726c946ea6079114d1e, 789a64c1409379d8fc7114eb66e92900ed1b14d8, b70193f597e1e693e88feaab5d79648317c54766, f351df48dc5a1ea4849ccceba9cb43fbf71cb168.

**Comment by pauloxnet** [comment]
Great!

**Comment by felixxm** [comment]
@pauloxnet Thanks!

**Comment by pauloxnet** [comment]
@felixxm you're welcome!
Can I ask yout to take a look on this 2 PR ( #901 #903 ) we sent in April during the Django Europe 2019 sprints ?

**Pull request merged**